phalcon/filter.zep

```diff
@@ -209,6 +218,17 @@ class Filter implements FilterInterface
 				}
 				return strtoupper(value);
 
+			case Filter::FILTER_BOOLEAN:
+				/**
+				 * Must break, as it may return a null value
+				 */
+				return is_bool($value) ? $value : null;
```

Comment:
Not sure what you are asking here.  Would you rather I use an if statement instead of the ternary operator?